### PR TITLE
Retreive comments

### PR DESCRIPTION
**Pull request by valvarez1218**
Adds the Comments class as well as the LoadComments servlet. Slightly changes the html and CSS so that the navigation bar and the comments list are distinguishable in the CSS. Adds the comments page.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -37,12 +37,23 @@ async function getArtist() {
...
+  fetch('/view-comments').then(response => response.json()).then((comments) => {
+    const commentElement = document.getElementById('comment-list');
+    comments.forEach((comment) => {
+      console.log(comment.message);
+      commentElement.appendChild(createCommentElement(comment));
+    })
+  });
+}
+
+function createCommentElement (comment) {
```
Comment (jmuzy-sps) [inline portfolio/src/main/webapp/script.js:50]: I might have missed it, where is `createCommentElement()` used?

**Review by jmuzy-sps — CHANGES_REQUESTED** [review]
Nie job! Just one quick comment! Otherwise you're good to submit this pull request and delete #12 and #13 since those changes are also in here

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (valvarez1218) [reply portfolio/src/main/webapp/script.js:50]: I used createCommentElement in the loadComments() function when I append to the comment-list

Comment (jmuzy-sps) [reply portfolio/src/main/webapp/script.js:50]: Ah, gotcha, thanks!

**Review by jmuzy-sps — APPROVED** [review]
Great job!

**Pull request merged**